diff -Naur adwaita-blue.orig/index.theme adwaita-blue/index.theme
--- adwaita-blue.orig/index.theme	2016-05-08 20:30:03.199309609 -0500
+++ adwaita-blue/index.theme	2016-05-08 20:30:53.278327692 -0500
@@ -1,5 +1,5 @@
 [Icon Theme]
-Name=Adwaita Blue
+Name=DeskOS
 Inherts=Adwaita,gnome,hicolor
 Example=folder
 Directories=16x16/actions,16x16/apps,16x16/mimetypes,16x16/places,16x16/status,22x22/actions,22x22/apps,22x22/mimetypes,22x22/places,22x22/status,24x24/actions,24x24/apps,24x24/mimetypes,24x24/places,24x24/status,32x32/actions,32x32/apps,32x32/mimetypes,32x32/places,32x32/status,48x48/actions,48x48/apps,48x48/mimetypes,48x48/places,48x48/status,64x64/actions,64x64/apps,64x64/mimetypes,64x64/places,64x64/status,96x96/actions,96x96/apps,96x96/mimetypes,96x96/places,96x96/status,256x256/actions,256x256/apps,256x256/mimetypes,256x256/places,256x256/status,scalable/actions,scalable/apps,scalable/mimetypes,scalable/places,scalable/status,
